README.md

```diff
@@ -4,49 +4,56 @@
 | [![Build status](https://ci.appveyor.com/api/projects/status/809xi9ukwo7sgsip?svg=true)](https://ci.appveyor.com/project/adam4813/tec-hem9u) | [![Build Status](https://semaphoreci.com/api/v1/trillek-team/tec/branches/master/shields_badge.svg)](https://semaphoreci.com/trillek-team/tec) |
 
 
-# Requirements
-TEC requires cmake 3.9 and a few libraries GLFW3, GLM, ASIO, Protobuf, GLEW, Lua, Bullet, Dear ImGui, Spdlog and OpenAL which can be installed most easily via [vcpkg](#vcpkg)
+## Requirements
+TEC requires cmake 3.9 and a few libraries GLFW3, GLM, ASIO, Protobuf, GLEW, Lua, Bullet, Dear ImGui, sol3, Spdlog and OpenAL which can be installed most easily via [vcpkg](#vcpkg)
 
-# VCPKG
-1. `git clone https://github.com/Microsoft/vcpkg.git` (If you already have **VCPKG**, go to [building](#building))
+## Building from Source
+### Part 1 (VCPKG/Requirements)
+1. `git clone https://github.com/Microsoft/vcpkg.git` (If you already have **VCPKG**, move onto Part 2))
 1. Navigate to the `vcpkg/` directory.
-	 1. Windows: `./bootstrap-vcpkg.bat` or Others: `./bootstrap-vcpkg..sh`
-	 1. **MacOS MAYBE** `sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /` if `bootstrap-vcpkg` fails see [here](https://donatstudios.com/MojaveMissingHeaderFiles) for more help
-	 1. **Linux** `apt-get install libgl1-mesa-dev xorg-dev libglu1-mesa-dev libxinerama-dev libxcursor-dev`
-	 1. **OPTIONAL** `./vcpkg integrate install`
-	 1. `./vcpkg install asio bullet3 glew glfw3 glm lua openal-soft protobuf zlib spdlog imgui sol2`
+#### Windows
+	`./bootstrap-vcpkg.bat` or Others: `./bootstrap-vcpkg..sh`
```

Comment:
Double `.` in `./bootstrap-vcpkg..sh`

Reply:
Best to just split this into windows vs others instead of putting this step under windows only.